UnitsNet.Tests/CustomCode/LengthTests.cs

```diff
@@ -32,6 +33,8 @@ public class LengthTests : LengthTestsBase
 
         protected override double FeetInOneMeter => 3.28084;
 
+        protected override double USSurveyFeetInOneMeter => 3.2808333333;
```

Comment:
I'm confused.  If mine is backwards, isn't the previous property "FeetInOneMeter" also backwards?  I just followed suit from FeetInOneMeter.  A U.S. Survey Foot is very close to an International Foot.